pkg/db-repo/orm.go

```diff
@@ -19,14 +19,21 @@ type OrmSettings struct {
 	Application string              `cfg:"application" default:"{app_name}"`
 }
 
+type OrmConfig struct {
+	DbName string `cfg:"db" default:"default"`
+}
+
 func NewOrm(config cfg.Config, logger mon.Logger) (*gorm.DB, error) {
-	dbClient, err := db.NewClient(config, logger, "default")
+	dbName := OrmConfig{}
+	config.UnmarshalKey("orm", &dbName)
```

Comment:
I would like a config key that is more related to the package name, like `db_repo`

Reply:
```
orm:
  db_repo: default

db:
  default:
    ...
```

Is this what you mean?